src/screens/Login.tsx

```diff
@@ -110,6 +114,11 @@ function Login() {
           {/* <Button label="Forgot password?" fullWidth variant="text" /> */}
         </form>
       </div>
+      <Snackbar
+        open={!isOnline}
+        noBottomMargin
+        message="You are not connected to a network. Please reconnect to log in."
+      />
```

Comment:
What do you think about the snackbar being on indefinitely if you're not online in login? IMO it's confusing otherwise because once the snackbar is gone there's no indication at all that the user is offline (even after a login fails)

Reply:
Good call -- that definitely makes more sense in this case.